ttnn/cpp/ttnn/operations/sliding_window/sliding_window.cpp

```diff
@@ -371,18 +371,17 @@ std::tuple<std::vector<std::vector<uint16_t>>, std::vector<std::vector<uint16_t>
                             flattened_remote_config);
 }
 
-std::vector<std::vector<uint16_t>> generate_sliding_window_op_config(const std::vector<uint32_t>& op_trace_metadata, const std::vector<std::pair<uint32_pair_t, uint32_pair_t>>& shard_boundaries, bool pad_tile, bool pad_last_core) {
+std::vector<std::vector<uint16_t>> generate_sliding_window_op_config(const std::vector<uint32_t>& op_trace_metadata, const std::vector<std::pair<uint32_pair_t, uint32_pair_t>>& shard_boundaries, bool pad_tile, bool pad_cores) {
     std::vector<std::vector<uint16_t>> sharded_input_top_left_indices;
     for(const auto& item : shard_boundaries) {
         const auto& [output_shard_start, output_shard_end] = item.first;
         const auto& [input_shard_start, input_shard_end] = item.second;
+        std::vector<uint16_t> local_top_left_indices;
         // sanity check
         if (output_shard_start >= op_trace_metadata.size()) {
-            // this core has no output
```

Comment:
keep the comment?

Reply:
oh sorry I had added functionality here which I then removed, I will add the comment back